components/Notifications.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import { Button, View } from 'react-native';
+import type { NativeStackScreenProps } from '@react-navigation/native-stack';
+
+type RootStackParamList = {
+  Home: undefined;
+  Profile: { userId: string };
+  Notifcations: { userId: string };
+  Settings: undefined;
+  Feed: { sort: 'latest' | 'top' } | undefined;
```

Comment:
to make a attribute optional
```suggestion
  Feed?: { sort: 'latest' | 'top' };
```